README.md

```diff
@@ -1,156 +1,128 @@
-Joomla! CMS™ [![Analytics](https://ga-beacon.appspot.com/UA-544070-3/joomla-cms/readme)](https://github.com/igrigorik/ga-beacon)
-====================
+[Browser Automated Tests for Joomla! CMS(covering Users and Content features - GSoC1 16)](https://summerofcode.withgoogle.com/projects/#5724182314745856)
+===
 
-Build Status
----------------------
-Travis-CI: [![Build Status](https://travis-ci.org/joomla/joomla-cms.svg?branch=staging)](https://travis-ci.org/joomla/joomla-cms)
-Jenkins: [![Build Status](http://build.joomla.org/job/cms/badge/icon)](http://build.joomla.org/job/cms/)
-
-What is this?
----------------------
-* This is a Joomla! 3.x installation/upgrade package.
-* Joomla's [Official website](https://www.joomla.org).
-* Joomla! 3.6 [version history](https://docs.joomla.org/Joomla_3.6_version_history).
-* Detailed changes are in the [changelog](https://github.com/joomla/joomla-cms/commits/master).
+Abstract
+---
+System development nowadays more than ever starts to look for automated test methods. There are several main drivers for this trend -need for faster design‐develop‐test‐analysis cycle, push for higher quality, increasing complexity of systems and their integration and last but not least ever‐rising costs of manual testing. Software testing itself is very wide field with topics ranging from compliance frameworks.
```

Comment:
there is a typo in `trend -need`

also the last sentence is missing `from compilance frameworks... to ....`, isn't it?